kbcstorage/workspaces.py

```diff
@@ -0,0 +1,112 @@
+"""
+Manages workspace requests to the API.
+
+Full documentation `here`.
+
+.. _here:
+    http://docs.keboola.apiary.io/#reference/workspaces/
+"""
+
+from kbcstorage.base import Endpoint
+
+
+def _make_body(mapping):
+    """
+    Given a dict mapping Keboola tables to aliases, construct the body of
+    the HTTP request to load said tables.
+
+    Args:
+        mapping(:obj:`dict`): Keys contain the full names of the tables to
+            be loaded (ie. 'in.c-bucker.table_name') and values contain the
+            aliases to which they will be loaded (ie. 'table_name').
```

Comment:
@Ogaday this is the API doc for the workspace load command.  It could probably use a full example though.  Is there a particular part that is missing for you?